### PR TITLE
Carma kappa test

### PR DESCRIPTION
**Pull request by manderberg**
Addition of carma_kappawetrtest.F90, its new benchmark file and the modified Makefiles in CARMA, CARMA/tests and CARMA/source to build the executable.

### CODE REVIEW

**Comment by fvitt** [comment]
Thanks for the PR.  I don't think we should merge this into ESCOMP/CARMA main until we merge the kappa_dev carma_base branch into ESCOMP/CARMA_base main.

**Review by cbardeen — APPROVED** [review]

**Review by fvitt — COMMENTED** [review]
Looks like you need to update source/base to the head of ESCOMP/carma_base master. Then if things look OK merge this branch into ESCOMP/CARMA master.

File: doc/ChangeLog [hunk 1]
```diff
@@ -1,5 +1,29 @@
 ===============================================================
 
+Tag name:
+Originator(s): manderberg
+Date: Dec 1, 2021
```
Comment (fvitt) [inline doc/ChangeLog:5]: Should update the date ?

**Review by fvitt — APPROVED** [review]
Looks like source/base now points to the head of ESCOMP/CARMA_base master.  Thanks.

File: doc/ChangeLog (hunk 1 above)
Comment (manderberg) [reply doc/ChangeLog]: Updated ChangeLog to reflect new date (Jan 19, 2022) and pushed it up to my carma_kappa_test branch.

**Comment by fvitt** [comment]
@zywshoon, do you need to review this pull request?  From my point of view, it is ready to merge into ESCOMP/CARMA master.

**Review by zywshoon — APPROVED** [review]

**Comment by fvitt** [comment]
@manderberg , I think you can go ahead and merge this pull request into master now that all the reviewers have approved.  Thanks for working on this.

**Pull request merged**